glide.yaml

```diff
@@ -4,17 +4,21 @@ import:
 - package: k8s.io/api
   version: kubernetes-1.14.0
 - package: k8s.io/apimachinery
-  version: kubernetes-1.14.0
+  repo:    https://github.com/openshift/kubernetes-apimachinery.git
+  version: origin-4.2-kubernetes-1.14.0
```

Comment:
Is there a reason why we need to use our fork? Is there a carry patch there that we need?

Reply:
We started needing `k8s.io/apimachinery/pkg/apis/meta/v1beta1.AddToScheme()` in `k8s.io/kubernetes/pkg/kubectl/scheme.init()` (in https://github.com/openshift/kubernetes.git's origin-4.2-kubernetes-1.14.0 branch).